### PR TITLE
Edit numerous Content/documentation typos and errors + update & optimize phrasing/descriptions

### PR DESCRIPTION
**Pull request by preserveddarnell**
Edited typos, optimized and rephrased descriptions for FAQ and other docs in contents.

### CODE REVIEW

File: content/faq/bounties.md [hunk 1]
```diff
@@ -15,12 +15,12 @@ To claim a bounty, please email [bounty@lbry.io](mailto:bounty@lbry.io). Please
 
 ### How do I get compensated?
 
-People who complete bounties are awarded LBRY credits. These credits will be transferred to a wallet address of your choice and are yours with no restrictions to do as you like.
+People who complete bounties are rewarded LBRY credits (LBC). These credits will be transferred to an LBRY wallet address of your choice and are yours, with no restrictions to do as you like.
```
Comment (jessopb) [inline content/faq/bounties.md]: I think people just pronounce LBRY "library".  So I wouldn't use 'an'.

File: content/faq/bounties.md [hunk 2]
```diff
@@ -15,12 +15,12 @@ To claim a bounty, please email [bounty@lbry.io](mailto:bounty@lbry.io). Please
...
 
-### I have questions about a specific bounty or want assistance in completing a bounty.
+### I have questions about a specific bounty or seek assistance in completing a bounty.
```
Comment (jessopb) [inline content/faq/bounties.md]: "I seek assistance" sounds sounds a little too fancy.

File: content/faq/bounties.md (hunk 1 above)
Comment (preserveddarnell) [reply content/faq/bounties.md]: A lot of people surprisingly don't know (or don't read "LBRY" as library) rather read each individual letter so I just thought an would be appropriate prior to the "L".

File: content/faq/content.md [hunk 3]
```diff
@@ -3,19 +3,19 @@ title: How does LBRY handle content and what am I allowed to upload?
...
+This guide provides answers to questions regarding what you may upload to the LBRY app and network and how it differs from current centralized/status quo systems. To learn more about reporting infringing or illegal content and DMCA procedures, please see our [DMCA article](https://lbry.io/faq/dmca). 
 
 ### What content can I legally upload to LBRY?
 
-You may upload content you created or own that does not infringe on the rights of others or is illegal, or other content you have permission to upload. This includes Public Domain content, Creative Commons or Open Source content shared according to the license requirements, and Copyrighted content where you have permission from the owner.
+You may upload content you created or own that does not infringe on the rights of others, is not illegal, or other content you have permission to upload. This includes Public Domain content, Creative Commons or Open Source content shared according to the license requirements, and Copyrighted content where you have permission from the owner.
```
Comment (jessopb) [inline content/faq/content.md]: This needs another pass. Perhaps:
You may upload legal content that you own or have obtained permission for as long as it does not infringe on the rights of others.

Comment (preserveddarnell) [reply content/faq/content.md]: Amended.

File: content/faq/dmca.md [hunk 4]
```diff
@@ -10,20 +10,20 @@ The Digital Millennium Copyright Act (DMCA) is United States Copyright Law writt
...
 
 ### Can I dispute a DMCA complaint?
 
-Yes, you may. We can't give you legal advice, so check with your local attorney or legal group about how to do this. There are also good resources online to learn about DMCA counter-notices. The [EFF has published an excellent guide](https://www.eff.org/issues/intellectual-property/guide-to-youtube-removals) about how to deal with legal issues facing online content creators. If you have done the research or feel the content was blocked in error, [please reach out to us](mailto:help@lbry.io). 
+Yes, you may. We can't give you legal advice, so check with your local attorney or legal group about how to do this. There are also good resources online to learn about DMCA counter-notices. The [EFF has published an excellent guide](https://www.eff.org/issues/intellectual-property/guide-to-youtube-removals) about how to deal with legal issues facing online content creators. If you have done the research or feel the content was blocked mistakenly, [please reach out to us](mailto:help@lbry.io).
```
Comment (jessopb) [inline content/faq/dmca.md]: I think "in error" fits better in this legal context.

Comment (preserveddarnell) [reply content/faq/dmca.md]: This specific section "Can I dispute a DMCA complaint" (I don't think) is being delivered in a legal context, it's more of a disclaimer for advisory I would say.

File: content/faq/referrals.md [hunk 5]
```diff
@@ -3,27 +3,27 @@ title: How does the referral program work?
...
+You will not receive rewards for each referral until those users join and use LBRY themselves (see below section regarding claimable referrals). This is to combat referral fraud and abuse.
 
 ### What is the criteria for a claimable referral?
 
-A claimable referral has the same definition as a Rewards eligible or Verified LBRY user. This means the referred user must have logged into the LBRY app and verified their account.  Users who redeemed rewards prior to the LBRY Open Beta (on 8/1/2017) were automatically grandfathered as verified users. Post LBRY Open Beta verification options can be seen [here](https://lbry.io/faq/identity-requirements). 
+A claimable referral has the same definition as a Rewards eligible or Verified LBRY user. This means the referred user must have logged into the LBRY app and verified their account.  Users who've redeemed rewards prior to the LBRY Open Beta (on 8/1/2017) were automatically "grandfathered" as verified users. Post-LBRY Open Beta verification options can be seen [here](https://lbry.io/faq/identity-requirements).
```
Comment (jessopb) [inline content/faq/referrals.md]: Not a big deal, but "Post-LBRY Open Beta verification options" sounds like LBRY is over. 
Perhaps "Current user verification options can be seen..."

File: content/faq/tipbot-discord.md [hunk 6]
```diff
@@ -6,31 +6,31 @@ order: 1
...
 
 ### Deposit
-Displays your Discord LBRY wallet address. Useful if you want to receive LBC's directly to your Discord wallet.
+Displays your Discord LBRY wallet address. Useful if you want to receive LBCs directly to your Discord wallet.
 **Example:**  
 `!tip deposit`
 
 ### Withdraw
-Use this to withdraw a chosen amount from your LBRY Discord wallet to another LBRY wallet such as the wallet in your LBRY app, Coinomi or to a LBC wallet on an exchange.  
+Use this to withdraw a chosen amount from your LBRY Discord wallet to another LBRY wallet such as the wallet in your LBRY app, Coinomi or to an LBRY wallet on an exchange.
```
Comment (jessopb) [inline content/faq/tipbot-discord.md]: "a LBRY" again?

File: content/faq/unable-to-stream.md [hunk 7]
```diff
@@ -4,17 +4,17 @@ category: troubleshooting
...
+Another common cause of this issue is the lack of access to UDP port 4444. LBRY employs UPnP (if enabled) in order to forward this port on your router automatically. If UPnP is disabled, it must either be enabled, or the port must be forwarded in your router configuration.  We have also seen instances where some ISPs may block UDP port 4444, and for LBRY to download content properly, this port must be manually configured - see steps below.  
 
 ### How to manually set the DHT (Content Network) port
 
-1. Shutdown LBRY and lbrynet-daemon (check running processes)
-2. Go to your [lbrynet folder](https://lbry.io/faq/lbry-directories)
-3. If you have a `daemon_settings.yml` file, add this line to it at the end(44444 is an example, can be changed): `dht_node_port: 44444`.
+1. Shutdown LBRY and lbrynet-daemon (check running processes on Task Manager.)
```
Comment (jessopb) [inline content/faq/unable-to-stream.md]: On a mac it's called "Activity Monitor", and lots of options in linux.
Perhaps "(check running processes using Task Manager on Windows or Activity Monitor on Mac)"

Comment (preserveddarnell) [reply content/faq/unable-to-stream.md]: amended

File: content/faq/dmca.md (hunk 4 above)
Comment (jessopb) [reply content/faq/dmca.md]: 'Unfairly' might work. To me, mistakenly sounds like one of those things you can technically do in English, but that sounds funny to me.

**Comment by jessopb** [comment]
I read all the changes. Aside from the discussion I commented on and debate on "a LBRY wallet" vs "an LBRY wallet" in other places, I'd say this is a great improvement.

File: content/faq/dmca.md (hunk 4 above)
Comment (preserveddarnell) [reply content/faq/dmca.md]: I feel like mistakenly would be better suited in terms of legal context, unfairly sounds more like something that was censored by administrators **not** in the context of legality.

File: content/faq/tipbot-discord.md (hunk 6 above)
Comment (preserveddarnell) [reply content/faq/tipbot-discord.md]: Amended.

File: content/faq/bounties.md (hunk 1 above)
Comment (preserveddarnell) [reply content/faq/bounties.md]: Amended.

**Comment by tzarebczan** [comment]
@preserveddarnell thanks again for the PR! Will follow up regarding LBC in appreciation for your effort via email by tomorrow.

**Pull request merged**